### ISSUE TITLE
Ambiguous TypeError with Reflect.getMetadata on v6.0.1 that prevents the project from starting

### ISSUE THREAD

**Issue by standemchuk** [nestjs/nest#1763]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

When launching an instance of nest.js (with express) after going through the migration notes from v5 to v6, I receive the following stack trace:
```
(node:17739) UnhandledPromiseRejectionWarning: TypeError
    at Object.getMetadata (/Users/standemchuk/projects/setel/nest-6-problem/node_modules/reflect-metadata/Reflect.js:355:19)
    at ListenerMetadataExplorer.scanForClientHooks (/Users/standemchuk/projects/setel/nest-6-problem/node_modules/@nestjs/microservices/listener-metadata-explorer.js:34:38)
    at scanForClientHooks.next (<anonymous>)
    at ListenersController.bindClientsToProperties (/Users/standemchuk/projects/setel/nest-6-problem/node_modules/@nestjs/microservices/listeners-controller.js:35:42)
    at items.forEach (/Users/standemchuk/projects/setel/nest-6-problem/node_modules/@nestjs/microservices/microservices-module.js:48:42)
    at Map.forEach (<anonymous>)
    at MicroservicesModule.bindClients (/Users/standemchuk/projects/setel/nest-6-problem/node_modules/@nestjs/microservices/microservices-module.js:46:15)
    at modules.forEach (/Users/standemchuk/projects/setel/nest-6-problem/node_modules/@nestjs/microservices/microservices-module.js:39:18)
    at Map.forEach (<anonymous>)
    at MicroservicesModule.setupClients (/Users/standemchuk/projects/setel/nest-6-problem/node_modules/@nestjs/microservices/microservices-module.js:37:17)
    at NestApplication.registerModules (/Users/standemchuk/projects/setel/nest-6-problem/node_modules/@nestjs/core/nest-application.js:65:38)
    at NestApplication.init (/Users/standemchuk/projects/setel/nest-6-problem/node_modules/@nestjs/core/nest-application.js:78:20)
    at NestApplication.listen (/Users/standemchuk/projects/setel/nest-6-problem/node_modules/@nestjs/core/nest-application.js:132:44)
    at exceptions_zone_1.ExceptionsZone.run (/Users/standemchuk/projects/setel/nest-6-problem/node_modules/@nestjs/core/nest-factory.js:106:40)
    at Function.run (/Users/standemchuk/projects/setel/nest-6-problem/node_modules/@nestjs/core/errors/exceptions-zone.js:8:13)
    at Proxy.args (/Users/standemchuk/projects/setel/nest-6-problem/node_modules/@nestjs/core/nest-factory.js:105:46)
    at /Users/standemchuk/projects/setel/nest-6-problem/src/main.ts:20:13
    at Generator.next (<anonymous>)
    at fulfilled (/Users/standemchuk/projects/setel/nest-6-problem/src/main.ts:4:58)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

## Expected behavior

The project should launch successfully.

## Minimal reproduction of the problem with instructions

Nothing too fancy, just a migration from v5 to v6 including the usage of platform-express

## What is the motivation / use case for changing the behavior?

No changing, just a fix required.

## Environment

<pre><code>
Nest version: 6.0.1

 
For Tooling issues:
- Node version: Tried on 8.12.0 and 10.15.3  
- Platform: Mac

Others:

</code></pre>
I checked `Reflect.getMetadata` input and for some reason instead of retrieving an object it actually receives a concatenated `host:port` value (`127.0.0.1:3016`). Don't know where that one is coming from, but surely not from my code, since I'm not concatenating these values anywhere.

**Comment by kamilmysliwiec** [comment]
Please, provide a minimal repository which reproduces your issue.

**Comment by standemchuk** [comment]
@kamilmysliwiec thanks for getting back to me on such short notice. Here's an example repo (I stripped most of my project stuff away from it): https://github.com/standemchuk/nest-6-issue-test-repo

Just run `npm install` and `npm run start:dev`. Let me know if you can reproduce it.

**Comment by kamilmysliwiec** [comment]
Fixed in 6.0.2 :) Thanks!
